storage/umass/Makefile

```diff
@@ -1,11 +1,21 @@
 #
 # Makefile for Phoenix-RTOS USB Mass Storage driver
 #
-# Copyright 2020 Phoenix Systems
+# Copyright 2020, 2024 Phoenix Systems
 #
 
-NAME := umass
+ifeq ($(TARGET_FAMILY),ia32)
+	UMASS_LIBS := libext2
+	UMASS_CFLAGS := -DUMASS_MOUNT_EXT2
```

Comment:
Why limit this feature to the ia32 only?

Reply:
Out of all targets that currently include umass (ia32 and imxrt*) only ia32 is built with libext2